docs/es-modules/all.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>Cloudinary Video Player</title>
```

Comment:
Not critical but the title should match the example name ....same comment for the second example as well.